.github/pull_request_template.md

```diff
@@ -27,5 +27,9 @@ Quality check list that cannot be automatically verified.
     - Regression test for the bug
         - [ ] Properly covers the bug
         - [ ] If an integration test is used, it could not be written as a unit test
+- Documentation
+    - [ ] A new CLI flag was introduced, is there a documentation that explains usage?
```

Comment:
is there **~~a~~** documentation

Reply:
Besides, we don't seem to have documentation about the CLI arguments (or I cannot find them)